.github/workflows/gcc.yml

```diff
@@ -21,12 +21,9 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Cache
-      uses: actions/cache@v3
+    - uses: friendlyanon/setup-vcpkg@v1
```

Comment:
It looks like the `Post Run friendlyanon/setup-vcpkg@v1` step is always failing.

Reply:
Still haven't figured out this one...

Reply:
The `Post Run` step sometimes succeeds, usually with `VS2019 (Debug)`. 

The key reported in a failing `Post Run friendlyanon/setup-vcpkg@v1` step differs from the one in the earlier `Run friendlyanon/setup-vcpkg@v1` step, but maybe that's OK.